### ISSUE TITLE
Consider adding a section for how to credit the Zig logo correctly

### ISSUE THREAD

**Issue by chaplchapl** [ziglang/logo#8]
Currently, there's no information who created the logo, how one should credit the creator and more. Please consider adding more information how to include appropriate attribution of the creator.

**Comment by DiskPoppy** [comment]
Yeah, pretty big oversight with no copyright information / owner, yet the license demands attribution.

**Comment by Tumbolisu** [comment]
According to Wikipedia, the author of the logo is @KeyboardRage .

https://commons.wikimedia.org/wiki/File:Zig_logo_2020.svg
https://github.com/KeyboardRage

**Comment by KeyboardRage** [comment]
> According to Wikipedia, the author of the logo is [@KeyboardRage](https://github.com/KeyboardRage) .
> 
> https://commons.wikimedia.org/wiki/File:Zig_logo_2020.svg https://github.com/KeyboardRage


Yeah I did the latest one. It was re-created from scratch inspired by the older one. Not sure what the full right attribution would be in this case
